custom/intrahealth/utils.py

```diff
@@ -363,20 +363,28 @@ def export_table(self):
         export_tables = []
         for individual_report in self.report_context['reports']:
             report_table = individual_report['report_table']
-            total_row = self._extract_value_from_report_table_row_value(report_table)
-            export_tables.append(self._export_table(report_table['title'], report_table['headers'],
-                                                    report_table['rows'], total_row))
+            extracted_rows = self._extract_all_rows_from_report(report_table)
+            extracted_total_row = self._extract_single_row_from_report(report_table['total_row'])
+            export_tables.append(self._export_table(report_table['title'],
+                                                    report_table['headers'],
+                                                    extracted_rows, extracted_total_row))
         return export_tables
 
+    def _extract_all_rows_from_report(self, report_table):
+        extracted_rows = []
+        for row in report_table['rows']:
+            extracted_rows.append(self._extract_single_row_from_report(row))
+        return extracted_rows
+
     @staticmethod
-    def _extract_value_from_report_table_row_value(report_table):
+    def _extract_single_row_from_report(list_to_extract_from):
```

Comment:
can this be called `row` instead of `list_to_extract_from`?

Reply:
how about `_sanitize_row`?

Reply:
those are way better names, thanks https://github.com/dimagi/commcare-hq/pull/23245/commits/6bfa1926fd18985c93518e56247dcff8c943dfdc